server/liveblog/themes/themes.py

```diff
@@ -157,148 +170,286 @@ def get_options(self, theme, options=None):
-                    # theme is needed a theme.json in the root folder.
-                    # save the public_url for that theme
-                    if name.endswith('theme.json'):
-                        theme['public_url'] = superdesk.upload.url_for_media(file_id).replace('theme.json', '')
-
-        previous_theme = self.find_one(req=None, name=theme.get('name'))
-        if previous_theme:
-            # retrieve the default settings of the current theme
-            default_theme_settings = get_resource_service('themes').get_default_settings(theme)
-            # retrieve the default settings of the previous theme
-            default_prev_theme_settings = get_resource_service('themes').get_default_settings(previous_theme)
-            # if there was a customization for the theme settings
-            if 'settings' in previous_theme:
-                options = []
-                old_theme_settings = {}
-                # save the existing theme
-                old_theme = previous_theme.copy()
-                # save the old/existing theme settings
-                if old_theme.get('options', False):
-                    options += old_theme.get('options')
-                for option in options:
-                    old_theme_settings[option.get('name')] = option.get('default')
-                    old_theme_settings.update(theme.get('old_theme_settings', {}))
-                # # initialize the theme settings values for the old theme based on the new settings
-                theme_settings = {}
-                # loop over theme settings
-                for key, value in old_theme_settings.items():
-                    # if the settings of previous theme are the same as for the current theme
-                    # we keep them in a new variable
-                    if value == default_prev_theme_settings[key]:
-                        theme_settings[key] = value
-                    # otherwise we keep the settings that are already on the theme
-                    else:
-                        default_theme_settings[key] = default_prev_theme_settings[key]
-                theme_settings.update(default_theme_settings)
-                # save the new theme settings
-                theme['settings'] = theme_settings
-            # theme settings at blog level
+        return results.get('created'), results.get('updated')
+
+    def _save_theme_file(self, name, theme):
+        with open(name, 'rb') as file:
+            # Set the content type
+            mime = magic.Magic(mime=True)
+            content_type = mime.from_file(name)
+            if content_type == 'text/plain' and name.endswith(tuple(CONTENT_TYPES.keys())):
+                content_type = CONTENT_TYPES[os.path.splitext(name)[1]]
+
+            final_file_name = os.path.relpath(name, CURRENT_DIRECTORY)
+
+            # TODO: Add version parameter to media_id() after merging related core-changes in amazon_media_storage
+            # and desk_media storage.
```

Comment:
also don't forget the
 ```
                    # don't use version for the `parent-iframe.js` script
                    # requering the version for this will be done from the client.
                    if name.endswith('parent-iframe.js'):
                        version = False
```